### PR TITLE
docs: update Tabs api

### PR DESCRIPTION
**Pull request by jesse3mh9a**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   docs: 4.6.0 changelog #26342,     #25138    |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by hengkx — APPROVED** [review]

File: components/tabs/index.zh-CN.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
 | keyboard | 开启键盘切换功能 | boolean | true |  |  |
 | renderTabBar | 替换 TabBar，用于二次封装标签头 | (props: DefaultTabBarProps, DefaultTabBar: React.ComponentClass) => React.ReactElement | - |  |  |
 | size | 大小，提供 `large` `default` 和 `small` 三种大小 | string | `default` |  |  |
-| tabBarExtraContent | tab bar 上额外的元素 | ReactNode \| {left?: ReactNode, right?: ReactNode} | - |  |  |
+| tabBarExtraContent | tab bar 上额外的元素 | ReactNode \| {left?: ReactNode, right?: ReactNode} | - | 4.6.0 |  |
```
Comment (xrkffgg) [inline components/tabs/index.zh-CN.md:37]: 版本后面那个 可以删了

Comment (jesse3mh9a) [reply components/tabs/index.zh-CN.md]: 好像发现, 之前`animated` (zh-CN) 书写上有点问题,  在这里我也改下

File: components/tabs/index.zh-CN.md [hunk 2]
```diff
@@ -23,26 +23,26 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
...
-| type | 页签的基本样式，可选 `line`、`card` `editable-card` 类型 | string | `line` |  |  |
-| onChange | 切换面板的回调 | function(activeKey) {} | - |  |  |
-| onEdit | 新增和删除页签的回调，在 `type="editable-card"` 时有效 | (targetKey, action): void | - |  |  |
-| onTabClick | tab 被点击的回调 | function(key: string, event: MouseEvent) | - |  |  |
-| onTabScroll | tab 滚动时触发 | function({ direction: `left` \| `right` \| `top` \| `bottom` }) | - | 4.3.0 |  |
+| 参数 | 说明 | 类型 | 默认值 | 版本 |
+| --- | --- | --- | --- | --- |
+| activeKey | 当前激活 tab 面板的 key | string | - |  |
+| addIcon | 自定义添加按钮 | ReactNode | - | 4.4.0 |
+| animated | 是否使用动画切换 Tabs, 仅生效于 \`tabPosition="top" | boolean \| { inkBar: boolean, tabPane: boolean } | `{ inkBar: true, tabPane: false }` |  |
```
Comment (xrkffgg) [inline components/tabs/index.zh-CN.md]:
```suggestion
| animated | 是否使用动画切换 Tabs, 仅生效于 \`tabPosition="top" | boolean \| { inkBar: boolean, tabPane: boolean } | { inkBar: true, tabPane: false } |  |
```

**Comment by jesse3mh9a** [comment]
刚有个地方没注意!
`tabBarExtraContent ` 只是`4.6.0`加了一个`type`, 之前版本 只支持type`ReactNode `.
这里要怎么描述呢?

**Comment by xrkffgg** [comment]
```diff
-- 4.6.0
++ object: 4.6.0 
```

再来个 PR

**Pull request merged**